samples/crypto/kmu_usage_nrf54l/README.rst

```diff
@@ -0,0 +1,149 @@
+.. _crypto_kmu_usage_nrf54l:
+
+Crypto: KMU usage (nRF54L series)
+#################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The KMU usage sample shows how to generate and use various persistent keys using the Platform Security Architecture (PSA) APIs.
+Persistent keys are stored in the Key Management Unit (KMU) of the device and retain their value between resets.
+The implementation of the PSA ITS API is provided by the Oberon PSA Crypto, which implements the driver wrapper role
```

Comment:
First sentence should mention the KMU as the persistent key sample also does persistent keys. The point here is to showcase how the KMU can be used to store persistent keys.

Reply:
Addressed.